app/models/notification.rb

```diff
@@ -0,0 +1,41 @@
+class Notification < ActiveRecord::Base
```

Comment:
I don't get your point.

> Why do we need to save them on the database?

So that we can persist them? And the user can see them when he gets back to the app? What would be the alternative?

As for the `seen` field, my idea would be that when a notification is seen, it would be destroyed (that endpoint is not implemented yet, since it wasn't needed for this part)